guides/pre-release-week.md

```diff
@@ -0,0 +1,68 @@
+
+- Create a draft PR in the [meilisearch-digitalocean](https://github.com/meilisearch/meilisearch-digitalocean) repository modifying the old MeiliSearch version **into the RC version**. This branch originating the PR should be named `bump-meilisearch-v*.*.*-test`. This branch is only created for test purposes and will be close at the end of the pre-release week.
+
+## 🧪 Testing
+
+- **Test manually the RC** with a Core team member or on your own. Should be done by all the Integration team members.
+- **Run all the SDKs automatic test suites against the RC** thanks to [this script](https://github.com/meilisearch/integration-scripts/tree/main/sdks-tests). If tests are not green, these should be justified.
+- **Test manually the DO image with the RC** on the `bump-meilisearch-v*.*.*-test` branch by following the [testing process steps](https://github.com/meilisearch/meilisearch-digitalocean/blob/master/CONTRIBUTING.md#test-before-releasing) **without merging the branch or submitting the image.**
+
+## 💬 Discussing
+
+- **Open a new issue in [integration-guide](https://github.com/meilisearch/integration-guides/issues/new)** (labeled `MeiliSearch bump`) about the changes in the next release of MeiliSearch that will impact one or several integration tools. See [an issue example](https://github.com/meilisearch/integration-guides/issues/52).<br>
+Based on the [MeiliSearch milestones](https://github.com/meilisearch/MeiliSearch/milestones), this issue should be divided into sub-sections for each feature/fix. These sub-sections explain in a concise way how the change impacts the integration tool(s) and how to fix/implement it if necessary.<br>
+If a change involves a huge implementation in the integration tools, a separated issue can be created and linked to the main issue.
+
+- **Discuss and approve the issue**. All the Integration team members are requested. The 🚀 emoji means approval.
+
+## 💻 Coding
+
+For each implementation or fix:
+
+- **Create a PR** up-to-date with `bump-meilisearch-v*.*.*` and pointing to `bump-meilisearch-v*.*.*` with all the changes.
+- **Make all the tests green**: the tests are running against the RC and not the latest stable version of MeiliSearch.<br>
+If they don't pass all, please explain the reason: another PR concerning other changes might indeed be needed.
+- **Ask for reviews**.
+- **Merge the PR** with the `Squash and Merge` button once you get an approval.<br>
+Don't use Bors: this tool would not run the tests against the RC so will not merge your PR.
+- Once the PR is merged into `bump-meilisearch-v*.*.*`, **update the main PR description** (the draft PR to merge `bump-meilisearch-v*.*.*` into `master`) with the topic of the just-merged PR.<br>
+Some tests still might fail on this main PR until the new release of MeiliSearch is out.
+
+## 🥳 After the MeiliSearch official release
```

Comment:
Is this really part of the "Pre-release week" section? It seems to me that it's out of the scope and could be a separated guide

Reply:
Hum, this is closely and only related to the pre-release week, so I would rather let it here for the moment. Not sure this is useful to create a separate guide "After the pre-release week".